src/helpers.ts

```diff
@@ -0,0 +1,233 @@
+import browserslist from "browserslist";
+import { Node, ESLintNode, BrowserListConfig } from "./types";
+
+/* eslint no-nested-ternary: off */
+
+/*
+3) Figures out which browsers user is targeting
```

Comment:
I think we should make it more clear what the numbers in the comments mean. When reading them individually, readers might not understand what they mean.